tabpy-server/server_tests/test_config.py

```diff
@@ -3,10 +3,7 @@
 import logging
 import tabpy_server
 
-try:
-    from unittest.mock import patch, call
-except ImportError:
-    from mock import patch, call
+from unittest.mock import patch, call
```

Comment:
Given bump to Py3.6 don't need to try...except here. There's lots of other compat code that can be removed in subsequent PRs